lib/filter/tests/test_filters_in_list.c

```diff
@@ -157,7 +157,11 @@ run_testcases(const char *top_srcdir)
 int
 main(int argc G_GNUC_UNUSED, char *argv[] G_GNUC_UNUSED)
 {
+#ifdef TOP_SRCDIR
```

Comment:
I like your approach, but I think it would be better, if the `autotools` based build would use the same approach.
Could you please update the `autotools` compile also ?